projects/igniteui-angular/src/lib/select/select.component.ts

```diff
@@ -109,6 +107,9 @@ export class IgxSelectComponent extends IgxDropDownComponent implements IgxSelec
     /** @hidden @internal */
     @ViewChild('input', { read: IgxInputDirective, static: true }) public input: IgxInputDirective;
 
+    /** @hidden @internal */
+    @ContentChild(IgxHintDirective, { read: ElementRef }) hintElement: ElementRef;
```

Comment:
Unless this is used in some template binding (don't think so) ti can be `private/protected` rather than hiding it from the API.